midealocal/devices/x40/message.py

```diff
@@ -153,12 +156,13 @@ def __init__(self, body):
 
 
 class Message40Response(MessageResponse):
-    def __init__(self, message):
-        super().__init__(message)
+    def __init__(self, message: bytes) -> None:
+        super().__init__(bytearray(message))
         if (
             self.message_type
             in [MessageType.set, MessageType.notify1, MessageType.query]
             and self.body_type == 0x01
         ):
             self.set_body(Message40Body(super().body))
         self.set_attr()
+        self.fields: dict[str, Any] = {}
```

Comment:
Can you please explain this? I didn't get where this came from.

Reply:
This is a variable used but not defined in init, which is a typing issue. Also the type itself was missing

Reply:
But you are resetting its value here

Reply:
Nope, I'm just initializing it, it's `__init__()` after all ;-)

Reply:
You're right, I'm sorry